src/DBQuerys.java

```diff
@@ -0,0 +1,195 @@
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+
+
+
+import java.util.Date;
+import java.util.Scanner;
+public class DBQuerys {
+    SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
+    PreparedStatement st;
+    PreparedStatement insertHotel;
+    ResultSet rs;
+    Statement stmt;
+    Scanner sc = new Scanner(System.in);
+    
+
+    public DBQuerys() {
+    }
+
+    //realiza o cadastro de novos hospedes
+    public void cadastroHospede(Connection conn) {
+        try {
+            
+                Date checkin = new Date();
+                stmt = conn.createStatement();
+                
+                System.out.println("Insira os dados do hospede");
+                System.out.print("Nome: ");
+                String name = sc.nextLine();
+                System.out.print("Email: ");
+                String emaill = sc.nextLine();
+                System.out.print("Telefone: ");
+                Long telefon = sc.nextLong();
+                String sql = "INSERT INTO hospedes (Nome, Email, Telefone) VALUES ( '" + name + "','" + emaill + "' , '"
+                        + telefon + "')";
+
+                System.out.println("Selecione o quarto pelo numero:");
+                
+                listar("quartos");
+
+
+                System.out.print("Digite o quarto: ");
+                int idQuarto = sc.nextInt();
+
+                
+                System.out.println("Digite o final da estadia dd/mm/yyyy: ");
+                String checkout = sc.next();
+                
+                stmt.executeUpdate(sql, Statement.RETURN_GENERATED_KEYS);
+
+                
+                
+                int idHospede = -1;
+                try (ResultSet rs = stmt.getGeneratedKeys()) {
+                    if (rs.next()) {
+                        idHospede = rs.getInt(1);
+                    }
+                }
+                
+                String checkinOk = sdf.format(checkin);
+                
+                
+                // Inserir um registro na tabela de reserva usando o idHospede como chave estrangeira
+                
+                insertHotel = conn.prepareStatement("INSERT INTO reserva (IdHospede, IdQuarto, CheckIn, CheckOut) VALUES (?, ?, ?, ?)");
+                insertHotel.setInt(1, idHospede);
+                insertHotel.setInt(2, idQuarto);
+                insertHotel.setString(3, checkinOk);
+                insertHotel.setString(4, checkout);
+                
+                insertHotel.executeUpdate();
+                    
+
+
+               
+            }catch (SQLException e) {
+                e.printStackTrace();          
+            } 
+    }
+    
+    public void listar(String tabela) {
+
+        try {
+
+            switch (tabela) {
+
+                case "quartos":
+
+                    st = DB.getConnection().prepareStatement("select * from quartos");
+                    rs = st.executeQuery("select * from quartos");
```

Comment:
```suggestion
                    rs = st.executeQuery();
```

Acredito que isso possa ter o mesmo resultado